cmd/system-probe/main.go

```diff
@@ -23,7 +23,9 @@ func main() {
 	flag.StringVar(&opts.checkClient, "client", "", "The client ID that the check will use to run")
 	flag.Parse()
 
-	runAgent()
+	exit := make(chan struct{})
+
```

Comment:
formatting nit: this empty line - you could also just inline this `exit` var in the function call below